test/models/king_test.rb

```diff
@@ -0,0 +1,50 @@
+require 'test_helper'
+
+class KingTest < ActiveSupport::TestCase
+
+	##### TEST VALID_MOVE METHOD ######
+
+  	test "check if move is valid (invalid case, no movement)" do
+		game = Game.create(:white_uid => 1, :black_uid => 1, :game_name => "New Game")
+		king = game.pieces.where(:pos_x => 3).where(:pos_y => 0).first
+		move = king.valid_move?([3, 0])
+		assert_not move	
+	end
+
+	test "check if move is valid (invalid case, off board)" do
+		game = Game.create(:white_uid => 1, :black_uid => 1, :game_name => "New Game")
```

Comment:
There is only one thing i'd caution against.  If you are create Game using Game.create it means you will likely still have those objects in your database when you run the tests again.  So you either A have to clear out your db each time. Or B remember to clean up after yourself.  I believe FactoryGirl takes care of that for you but you don't have to use FactoryGirl